jpsreport/Analysis.cpp

```diff
@@ -264,6 +264,7 @@ std::map<int, polygon_2d> Analysis::ReadGeometry(
 
         if(geoPoly.count(area->_id) == 0) {
             Log->Write("ERROR: \t No polygon containing the measurement id [%d]", area->_id);
+            exit(EXIT_FAILURE);
```

Comment:
I suggest that we do not use exit not during the run of the program but instead throw exceptions. These exception should be caught on top level. Similar to jpscore (see main.cpp)